<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Address</class>
 <widget class="QDialog" name="Address">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>File Server Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileport_lineEdit">
      <property name="text">
       <string>8002</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>271</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Chat Server Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="chatport_lineEdit">
      <property name="text">
       <string>8001</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>271</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_lineEdit">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
